lib/Net/Async/Redis.pm

```diff
@@ -1161,11 +1161,12 @@ sub execute_command {
         $self->{connection_in_progress}
         ? Future->done
         # Are we the owner of a current MULTI transaction?
-        : $self->{_is_multi}
-        ? $self->connected
-        : Future->wait_all(
```

Comment:
Here `wait_all` isn't enough. if not connected, then `$self->{pending_multi}` should not be executed.

Reply:
the change made here, seems different from your CPAN update? 
https://github.com/regentmarkets/cpan/pull/2842/files

Reply:
yes. That version is a simple version. the reason is here: https://redmine.deriv.cloud/issues/65123#How